git_machete/cli.py

```diff
@@ -556,6 +563,13 @@ def launch(orig_args: List[str]) -> None:
                     machete_client.annotate(branch, parsed_cli.annotation_text)
                 else:
                     machete_client.print_annotation(branch)
+        elif cmd == "clean":
+            machete_client.read_definition_file()
+            machete_client.delete_untracked(opt_yes=cli_opts.opt_yes)
```

Comment:
Ha! I've seen one nuance that can be improved: branches become untracked after `git fetch --prune`... but this is only executed inside `machete_client.checkout_github_prs`, which happens later in the flow ☹️

This caused that after running `git machete github sync`, I've ended up with an untracked managed branch :///

Reply:
@PawelLipski I see, in that case moving `machete_client.delete_untracked(opt_yes=cli_opts.opt_yes)` after `machete_client.checkout_github_prs(pr_nos=[], my_opened_prs=True)` should fix it, wdyt?

Reply:
Okay! let's give it a shot, pls update `github sync` and the ordering in the relevant docs as well